android/java/org/chromium/chrome/browser/download/settings/BraveDownloadSettings.java

```diff
@@ -54,6 +69,12 @@ private void updateDownloadSettings() {
         boolean automaticallyOpenWhenPossible = ContextUtils.getAppSharedPreferences().getBoolean(
                 BravePreferenceKeys.BRAVE_DOWNLOADS_AUTOMATICALLY_OPEN_WHEN_POSSIBLE, true);
         mAutomaticallyOpenWhenPossiblePref.setChecked(automaticallyOpenWhenPossible);
+
+        boolean downloadProgressNotificationBubble =
+                ContextUtils.getAppSharedPreferences().getBoolean(
+                        BravePreferenceKeys.BRAVE_DOWNLOADS_DOWNLOAD_PROGRESS_NOTIFICATION_BUBBLE,
+                        false);
```

Comment:
Are we sure we want it to be false by default?

Reply:
Yes, it's mentioned in the ticket and so many user's complained about it in the community as well as reviews.